inline_scan.sh

```diff
@@ -195,12 +191,8 @@ get_and_validate_analyzer_options() {
             display_usage_analyzer >&2
             exit 1
         fi
-    elif [[ ! "${g_flag:-}" ]] && [[ ! "${d_flag:-}" ]] && [[ ! "${m_flag:-}" ]]; then
-        printf '\n\t%s\n\n' "ERROR - must provide an image digest, manifest, or specify -g to generate a digest" >&2
-        display_usage_analyzer >&2
-        exit 1
-    elif [[ "${g_flag:-}" ]] && ([[ "${d_flag:-}" ]] || [[ "${m_flag:-}" ]]); then
-        printf '\n\t%s\n\n' "ERROR - cannot generate digest when a manifest or digest is provided" >&2
+    elif [[ "${m_flag:-}" ]]; then
+        printf '\n\t%s\n\n' "ERROR - cannot generate digest when a manifest is provided" >&2
```

Comment:
If I'm not missing something, providing `-m` option now triggers always an error, so this must be fixed.